builtin/logical/pki/path_tidy.go

```diff
@@ -1459,6 +1496,21 @@ func (b *backend) pathConfigAutoTidyWrite(ctx context.Context, req *logical.Requ
 		}
 	}
 
+	if revokedSafetyBufferRaw, ok := d.GetOk("revoked_safety_buffer"); ok {
+		if config.RevokedSafetyBuffer == nil {
+			config.RevokedSafetyBuffer = new(time.Duration) // Allocate memory for pointer
+		}
+		*config.RevokedSafetyBuffer = time.Duration(revokedSafetyBufferRaw.(int)) * time.Second
+		if *config.RevokedSafetyBuffer < 1*time.Second {
+			return logical.ErrorResponse(fmt.Sprintf("revoked_safety_buffer must be at least one second; got: %v", revokedSafetyBufferRaw)), nil
+		}
+	} else {
+		// If no explicit value for revoked_safety_buffer, default it to SafetyBuffer
+		if config.RevokedSafetyBuffer == nil {
+			config.RevokedSafetyBuffer = &config.SafetyBuffer
```

Comment:
I would probably clone the value here again. I know we're just writing it to storage but I'd probably be safe and prevent accidental modifications from causing issues.

Reply:
Good idea! Updated to prevent potential issues from accidental modifications.